src/main/metabase/api.ts

```diff
@@ -259,10 +265,13 @@ export async function hasUserSetup(): Promise<boolean> {
 }
 
 export async function syncMetabaseSchema(): Promise<void> {
-  const databases = (await get('/api/database')).data as Array<{ id: number; is_sample: boolean }>;
+  const databases = (await get('/api/database')).data as Array<Database>;
+  const anonymizedAccessDbId = findAnonymizedAccessDbId(databases);
+
   for (const db of databases) {
-    if (!db.is_sample) {
+    if (!db.is_sample && db.id !== anonymizedAccessDbId) {
       await post(`/api/database/${db.id}/sync_schema`, {});
     }
   }
+  await post(`/api/database/${anonymizedAccessDbId}/sync`, {});
```

Comment:
It sounds weird that `sync` does only the schema sync and `schema_sync` does the schema sync + analysis...